graylog2-web-interface/webpack.config.js

```diff
@@ -96,6 +96,7 @@ if (TARGET === 'start') {
     },
     plugins: [
       new webpack.HotModuleReplacementPlugin(),
+      new webpack.NamedModulesPlugin(),
```

Comment:
Next time it would be very helpful adding a link or a little description of what a module does. I spent quite some time trying to figure out what `NamedModulesPlugin` does.